messaging/src/main/java/org/axonframework/eventhandling/TrackingEventProcessor.java

```diff
@@ -415,6 +415,14 @@ public boolean isError() {
         return state.get() == State.PAUSED_ERROR;
     }
 
+    /**
+     * Returns the {@link StreamableMessageSource} this processor is using
+     * @return {@link StreamableMessageSource}
+     */
+    public StreamableMessageSource<TrackedEventMessage<?>> getMessageSource() {
```

Comment:
Just to be semantically correct, the return type should be `StreamableMessageSource<? extends TrackedEventMessage<?>>`. It's not that way in the builder yet, but that is a backwards compatible change and should be changed at some point.